docs/catalog/sinks/jira.rst

```diff
@@ -53,13 +58,23 @@ Save the file and run
 
     helm upgrade robusta robusta/robusta --values=generated_values.yaml
 
-You should now get playbooks results in Jira! Example is shown below:
-
-    .. image:: /images/jira_example.png
-      :width: 1000
-      :align: center
+Try the example below to make sure you are recieving alert notifications on Jira. 
 
 
 .. note::
 
     2-way interactivity (``CallbackBlock``) isn't implemented yet.
+
+Test your Jira integration
+-------------------------------
+
+The example below creates a crashing pod which triggers ``KubePodCrashLooping`` event. 
```

Comment:
I would explain a little more.

`The command below creates a crashing pod which triggers the ``KubePodCrashLooping`` alert. This will cause a Jira ticket to be opened when using the above example.`